signals/subscription_all_categories.py

```diff
@@ -0,0 +1,36 @@
+import swapper
+
+from django.dispatch import receiver
+
+from rest_framework import status
+from rest_framework.response import Response
+
+from django.db.models.signals import (
+    post_save
+)
+
+from categories.models import (
+    UserSubscription,
+    Category
+)
+
+@receiver(post_save, sender = swapper.get_model_name('kernel', 'Person'))
+def subscription_all_categories(sender, instance, **kwargs):
+    
```

Comment:
Instead of line break, here we start comment with """ and the documentation comes on the next line.